From 3f2162fb08b070059042eee04758b347aeebc3c4 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sat, 9 Jun 2018 16:20:05 +0300
Subject: [PATCH 5/6] power: Remove interaction_with_handle

* The usage of this code was removed in commit
  0c226b7921b421dd24363f15aefef1236aa25287

Change-Id: Idf55eb17ae02929b9518c2a81e9de3d022ec0ff9
---
 utils.c | 17 +----------------
 utils.h |  1 -
 2 files changed, 1 insertion(+), 17 deletions(-)

diff --git a/utils.c b/utils.c
index 13dbccc..f9b2773 100644
--- a/utils.c
+++ b/utils.c
@@ -254,22 +254,7 @@ void interaction(int duration, int num_args, int opt_list[])
 #endif
 }
 
-int interaction_with_handle(int lock_handle, int duration, int num_args, int opt_list[])
-{
-    if (duration < 0 || num_args < 1 || opt_list[0] == 0)
-        return 0;
-
-    if (qcopt_handle) {
-        if (perf_lock_acq) {
-            lock_handle = perf_lock_acq(lock_handle, duration, opt_list, num_args);
-            if (lock_handle == -1)
-                ALOGE("Failed to acquire lock.");
-        }
-    }
-    return lock_handle;
-}
-
-//this is interaction_with_handle using perf_hint instead of
+//this is interaction using perf_hint instead of
 //perf_lock_acq
 int perf_hint_enable(int hint_id , int duration)
 {
diff --git a/utils.h b/utils.h
index 5af4d16..83f815e 100644
--- a/utils.h
+++ b/utils.h
@@ -42,7 +42,6 @@ void undo_hint_action(int hint_id);
 void undo_initial_hint_action();
 void release_request(int lock_handle);
 void interaction(int duration, int num_args, int opt_list[]);
-int interaction_with_handle(int lock_handle, int duration, int num_args, int opt_list[]);
 int perf_hint_enable(int hint_id, int duration);
 
 long long calc_timespan_us(struct timespec start, struct timespec end);
-- 
2.17.1

